### PR TITLE
Fr/feature/why hardhat block

### PR DESCRIPTION
**Pull request by RavenInd**
1. Corrected desktop layout logic
2. Corrected CTA styles
3. Implemented WhyHardhatBlock

### CODE REVIEW

File: docs/src/pages/_document.tsx [hunk 1]
```diff
@@ -3,6 +3,9 @@ import { Html, Head, Main, NextScript } from 'next/document';
 export default function Document() {
   return (
     <Html>
+      <Head>
+        <link rel="shortcut icon" href="/static/favicon.ico" />
```
Comment (usulpro) [inline docs/src/pages/_document.tsx]: Let's move it to SEO component

File: docs/src/pages/index.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import WhyHardhatBlock from '../components/landingBlocks/WhyHardhatBlock';
 
 const Home: NextPage = () => {
   return (
-    <LandingLayout seo={{ title: 'Hardhat' }}>
+    <LandingLayout seo={{ title: 'Hardhat | Ethereum development environment for professionals by Nomic Foundation' }}>
```
Comment (usulpro) [inline docs/src/pages/index.tsx]: Good. You can also add a description the same way

File: docs/src/components/landingBlocks/HeroBlock.tsx [hunk 3]
```diff
@@ -11,7 +11,7 @@ const { HeroPetsImage, HeroGraphicsImage } = Images;
 
 const content = {
   title: 'Ethereum development environment for professionals',
-  tagline: 'Flexible. Extensible. Fast.',
+  tagline: 'Flexible. Composable. Fast.',
```
Comment (usulpro) [inline docs/src/components/landingBlocks/HeroBlock.tsx]: it worth to confirm this with the clients

File: docs/src/components/landingBlocks/WhyHardhatBlock.tsx [hunk 4]
```diff
@@ -61,12 +48,41 @@ const Title = styled.h2`
...
+`;
+
 const WhyHardhatBlock = ({ content }: Props) => {
   return (
     <Section>
       <Container>
+        <TopBrackets />
         <Title>{content.title}</Title>
-        {/* <FeatureCard content={content.featureCardOne} isReversed={true} /> */}
+        <FeatureCard content={content.featureCardOne} isReversed={true} />
```
Comment (usulpro) [inline docs/src/components/landingBlocks/WhyHardhatBlock.tsx]: These FeatureCards should be added on the page level. If the reason in brackes - let's create a wrapper component accepting arbitrary children

File: docs/src/config.ts [hunk 5]
```diff
@@ -51,3 +59,62 @@ export enum Tools {
   NETWORK = 'NETWORK',
   VS_CODE = 'VS_CODE',
 }
+
+export const FeatureCards = {
```
Comment (usulpro) [inline docs/src/config.ts]: TODO: we need to find a better place where to keep this content

**Review by usulpro — CHANGES_REQUESTED** [review]
looks good but see the comments please

File: docs/src/pages/_document.tsx (hunk 1 above)
Comment (RavenInd) [reply docs/src/pages/_document.tsx]: Done

File: docs/src/pages/index.tsx (hunk 2 above)
Comment (RavenInd) [reply docs/src/pages/index.tsx]: Done

File: docs/src/components/landingBlocks/HeroBlock.tsx (hunk 3 above)
Comment (RavenInd) [reply docs/src/components/landingBlocks/HeroBlock.tsx]: Ok. I'll do it at the next call with the customer. I don't think it's urgent

File: docs/src/components/landingBlocks/WhyHardhatBlock.tsx (hunk 4 above)
Comment (RavenInd) [reply docs/src/components/landingBlocks/WhyHardhatBlock.tsx]: Done

File: docs/src/config.ts (hunk 5 above)
Comment (RavenInd) [reply docs/src/config.ts]: We can create a folder config instead of config file or we can create a folder with default content

Comment (usulpro) [reply docs/src/config.ts]: let's move it into the page file itself

**Review by usulpro — APPROVED** [review]
fix this https://github.com/focusreactive/hardhat/pull/6#discussion_r837694647
update branch, check linting & preview

**Pull request merged**